projectDocs/translating/readme.md

```diff
@@ -6,7 +6,8 @@ There are several tasks to be done, and this document hopes to give you an overv
 
 ## Translation Mailing List
 
-Translators should subscribe to the [NVDA translations mailing list](https://groups.io/g/nvda-translations) hosted at Groups.IO.
+To monitor changes to symbols and character description files, translators should subscribe to the [nvda-l10n](https://groups.google.com/a/nvaccess.org/g/nvda-l10n/about) mailing list.
```

Comment:
Should we indicate that it's a read-only list, if it actually is, as the API mailing list is? I.e. users cannot send e-mail on this list.